Modules/Scripted/SampleData/SampleData.py

```diff
@@ -300,7 +300,11 @@ def setCategoriesFromSampleDataSources(categoryLayout, dataSources, logic):
         iconPath = os.path.join(os.path.dirname(__file__).replace('\\', '/'), 'Resources', 'Icons')
         mainWindow = slicer.util.mainWindow()
         if mainWindow:
-            iconSize = qt.QSize(int(mainWindow.width / 8), int(mainWindow.height / 6))
+            # Set thumbnail size from default icon size. This results in toolbutton size that makes
+            # two columns of buttons fit into the size of the Welcome module's minimum width
+            # on screens with a various resolution and scaling (see qt.QDesktopWidget().size,
+            # desktop.devicePixelRatioF(), qt.QDesktopWidget().physicalDpiX())
+            iconSize = qt.QSize(int(mainWindow.iconSize.width() * 6), int(mainWindow.iconSize.height() * 4))
```

Comment:
Thanks for fixing this 👌🙏

Are the multipliers `6` and `4` required?

Reply:
Without the multipliers the thumbnails would be as small as the toolbar icons - too small to be useful:

![image](https://user-images.githubusercontent.com/307929/207125443-21570feb-d803-4330-969c-4275e2ef9061.png)

Reply:
I understand now. Thanks for clarifying.